seqr/utils/elasticsearch/es_utils_tests.py

```diff
@@ -2403,6 +2404,40 @@ def test_get_es_variants_index_alias(self):
         self.assertDictEqual(urllib3_responses.call_request_json(index=0), {
             'actions': [{'add': {'indices': [INDEX_NAME, SECOND_INDEX_NAME, SV_INDEX_NAME], 'alias': INDEX_ALIAS}}]})
 
+    @urllib3_responses.activate
+    def test_get_es_variants_search_index_alias(self):
+        search_model = VariantSearch.objects.create(search={
+            'annotations': {'frameshift': ['frameshift_variant']},
+            'inheritance': {'mode': 'de_novo'},
+        })
+        results_model = VariantSearchResults.objects.create(variant_search=search_model)
+        results_model.families.set(Family.objects.filter(guid__in=['F000011_11']))
+
+        setup_search_responses()
+        urllib3_responses.add_json('/{}/_mapping'.format(SECOND_INDEX_NAME), {
+            INDEX_NAME: {'mappings': INDEX_METADATA[SECOND_INDEX_NAME]},
+            NO_LIFT_38_INDEX_NAME: {'mappings': INDEX_METADATA[SECOND_INDEX_NAME]},
+        })
+
+        get_es_variants(results_model, num_results=2)
+
+        expected_search = {
+            'start_index': 0, 'size': 2, 'filters': [ANNOTATION_QUERY, {
+                'bool': {'must': [
+                    {'bool': {'should': [
+                        {'term': {'samples_num_alt_1': 'NA20885'}},
+                        {'term': {'samples_num_alt_2': 'NA20885'}},
+                    ]}}
+                ],
+                '_name': 'F000011_11'
+            }}]
+        }
+        self.assertExecutedSearches([
+            dict(index=NO_LIFT_38_INDEX_NAME, **expected_search),
+            dict(index=INDEX_NAME, **expected_search),
+        ])
+        _set_cache('index_metadata__{}'.format(SECOND_INDEX_NAME), None)
+
```

Comment:
no, since the results are mocked and theres no special processing for alias search theres no point in testing the results. This is consistent with many of the tests. in this file